**Question 1**

What is the typical engagement rate we can expect? What’s the likelihood that we can achieve a 15% engagement rate?

**1. Import the data set**

In [ ]:
import pandas as pd

# Read the data from the Google Sheet
df = pd.read_excel("/content/social_data.xlsx")

df.head()

Published Date Account Account Type          Campaign Name  \
0 2023-03-31 19:55:43.192    CSGO      TWITTER                   N/A    
1 2023-03-31 19:49:35.243    CSGO      TWITTER                   N/A    
2 2023-03-31 19:49:33.535    CSGO      TWITTER                   N/A    
3 2023-03-31 19:49:31.583    CSGO      TWITTER                   N/A    
4 2023-03-31 19:43:42.590    CSGO      TWITTER  Community Engagement    

   Total Impressions  Total Engagements Media Type  
0                  0                  0       Text  
1                  0                  0       Text  
2                  0                  0       Text  
3                  0                  0       Text  
4               9517               1215      Video

**2. Calculate the required values**

In [ ]:
# Calculate the average engagement rate
avg_engagement_rate = df["Total Engagements"].mean() 

# Calculate the likelihood of achieving a 15% engagement rate
likelihood_of_15_engagement_rate = df["Total Engagements"].between(15, 100).sum() / len(df)

# Print the results
print(f"The average engagement rate is: ", avg_engagement_rate)
print(f"The likelihood of achieving a 15% engagement rate is: ", likelihood_of_15_engagement_rate)


The average engagement rate is:  947.3906294912331
The likelihood of achieving a 15% engagement rate is:  0.22305260132221902


This indicates that company's **Total Average Engagement** is **947.39**. The **probability** of obtaining a 15% engagement rate is **0.22 out of 1**. This indicates that while a 15% engagement rate is theoretically feasible, it is unlikely. To attain this level of involvement, you will need to devote a considerable amount of time and money.


Additional advice for raising engagement rates is provided below:

*   Produce compelling material that speaks to your intended audience.
*   Consistently and often post.
*   Run gifts and contests.
*   Activate user sharing of your content.








